### PR TITLE
Separate criterion benches

### PR DESCRIPTION
**Pull request by gabrielbosio**
**NOTE:** To merge this, #144 has to be merged.

### CODE REVIEW

**Review by diegokingston — APPROVED** [review]

File: src/cairo/air.rs [hunk 1]
```diff
@@ -221,6 +225,223 @@ impl PublicInputs {
...
+            ap_final,
+            range_check_min,
+            range_check_max,
+            memory_segments,
+            public_memory,
+            num_steps,
+        })
+    }
+}
+
```
Comment (Oppen) [inline src/cairo/air.rs:444]: This bit looks like it belongs to a different PR.

File: src/main.rs
[the remark this answers is not in the document]
Comment (gabrielbosio) [reply src/main.rs:1]: Yes, it's branched from #144.

**Review by entropidelic — APPROVED** [review]
